web/modules/custom/self_deposit/js/self_deposit.js

```diff
@@ -0,0 +1,14 @@
+Drupal.behaviors.self_deposit_debounce = {
+    attach: function (context, settings) {
+  
+      // Attach a click listener to the submit button.
+      var form = document.getElementById("webform-submission-self-deposit-add-form");
+      var sBtn = document.getElementById('edit-actions-wizard-next');
```

Comment:
This element doesn't appear to exist by the time we submit the form. My guess is there is JS schenanigans that replace it. Dev tools shows this element as having the ID "edit-actions-submit". Also, that form ID doesn't exist, but you should be able to get the form by using `sBtn.form`. That said, I attempted replacing `form.submit();` with `sBtn.form.submit();`, but that didn't appear to work.